src/common/utils/getExportData.js

```diff
@@ -0,0 +1,84 @@
+import {
+  flatten,
+  uniq,
+} from 'lodash';
+
+import {
+  CONFIG_ADDRESSES,
+  MODULE_TENANT,
+} from '../../components/Utils/const';
+import { createExportReport } from './createExportReport';
+import { getAddresses } from './getAddresses';
+import { fetchExportDataByIds } from './fetchExportDataByIds';
+
+export const getExportData = async (
```

Comment:
IMHO, you definitely have a lack of domain driven file structure, it looks like you need a component with corresponding code in it instead of moving all functions in common.